src/Microsoft.Graph.Core/CoreConstants.cs

```diff
@@ -41,6 +41,12 @@ public static class Headers
 
             /// Feature Flag
             public const string FeatureFlag = "FeatureFlag";
+
+            /// Contnet type header name
+            public const string ContentTypeHeaderName = "Content-Type";
+
+            /// JSON content type value
+            public const string JsonContentType = "application/json";
```

Comment:
maybe worth looking at https://stackoverflow.com/a/54650792/3808675 or something equivalent instead of repeating constants

Reply:
@baywet We are targeting .NetStandard 1.1 which doesn't have `System.Net.Mime` constants. I'll however borrow a leaf for their [implementation](https://github.com/dotnet/corefx/blob/master/src/System.Net.Mail/src/System/Net/Mime/MediaTypeNames.cs) to group our header names & values.